packages/frontend-2/components/user/profile/EditDialog.vue

```diff
@@ -34,10 +40,21 @@ const props = defineProps<{
 }>()
 
 const { result } = useQuery(profileEditDialogQuery)
+
 const user = computed(() => result.value?.activeUser)
 
 const isOpen = computed({
   get: () => !!(props.open && user.value),
   set: (newVal) => emit('update:open', newVal)
 })
+
+const developerSettingsButton = {
```

Comment:
Non-reactive const. In this case there's nothing that would require reactivity here, but it's still good practice to wrap this with a computed or something, in case someone uses something reactive inside this button definition in the future